.github/workflows/quarkus.yml

```diff
@@ -0,0 +1,335 @@
+name: Nightly quarkus tests
+
+on: [push]
+
+env:
+  # Workaround testsuite locale issue
+  LANG: en_US.UTF-8
+  JVMCI_VERSION: jvmci-20.1-b01
+  JDK_VERSION: 11.0.6
+  JDK_VERSION_SUFFIX: "+9"
+  DB_USER: hibernate_orm_test
+  DB_PASSWORD: hibernate_orm_test
+  DB_NAME: hibernate_orm_test
+  NATIVE_TEST_MAVEN_OPTS: "-B -Dtest-postgresql -Dtest-elasticsearch -Dtest-keycloak -Dtest-dynamodb -Dtest-mysql -Dtest-mariadb -Dmariadb.url='jdbc:mariadb://localhost:3308/hibernate_orm_test'  -Dtest-mssql -Dtest-vault -Dtest-neo4j -Dnative-image.xmx=5g -Dnative -Dformat.skip install"
+  MX_GIT_CACHE: refcache
+
+jobs:
+  build-quarkus-and-graalvm:
+    name: Nightly quarkus and GraalVM build
```

Comment:
when done on push this isn't really nightly ;)

Reply:
I believe Foivos commented on that earlier: "This github action will be ultimately set to run nightly. It is currently set to run on push for testing purposes till it gets reviewed."

Reply:
sorry I missed that.

Reason for the nightly is that it takes too long to verify ? I would assume run on pull-requests would be useful or ?

Reply:
I just assumed we wanted it Nightly because our initial discussion was about nightly builds.
Running on pull-requests would certainly prevent breaking changes from making it to `master`. 

The whole workflow needs something less than 1h and 30m at the moment.

Reply:
sorry if I made it sound initially as a nightly build - that was not my intention. 

Intention was to have *something* on graalvm side that can be used as early warning signs if some change in graalvm master affects quarkus (and vice versa) - preferably that would be on `push` but if deemed too long/too flakey nightly builds would do.

My preference is to have it at least on pull-requests coming in and on push to master (or nightly) would at least catch those things that are not coming in via pull-requests too.

Reply:
I propose to start with a nightly task that tests the tip of `graal` with the latest stable release of Quarkus.
This would make our tests more stable and reproducible.
If we see that this is not enough, we can easily change policy.

Reply:
https://github.com/oracle/graal/commit/55990119a23baeadf3967ae3565aea82069d5352 makes it build using the latest github release of quarkus instead of the tip (this includes candidate releases, but I think this is OK, please let me know if you disagree).

I am letting it run on push till we _finalize_ this PR.

Reply:
Thanks!

Reply:
@ansalond Just a reminder that before merging I need to resolve the above ^^

Reply:
@ansalond I have now configured it to run nightly at 3am and when changes affecting the workflow are pushed.
Note that this workflow tests latest graal with the latest quarkus release from https://github.com/quarkusio/quarkus/releases (no longer testing pre-releases)